0-buggy.cpp

```diff
@@ -1,9 +1,23 @@
-#include <cstdio>
+using namespace std;
+#include <iostream>
+#include <stdio.h>
+int max_total_sum = 2000;

```

Comment:
Add empty line for readability
```suggestion

int max_total_sum = 2000;
```